src/openfl/_internal/renderer/opengl/GLVAORenderHelper.hx

```diff
@@ -51,6 +51,7 @@ class GLVAORenderHelper {
 			renderSession.shaderManager.updateShader (shader);
 			shader.__skipEnableVertexAttribArray = false;
 			
+			bitmapData.__vaoContext = vaoContext;
```

Comment:
Shouldn't we check here if the context has changed and recreate the vao even if there's one?

> PS This makes me wonder if it's possible to use same `BitmapData` across multiple GL contexts? Theoretically it should be possible, but I guess not really, because texture is also bound to the context it was created for...

Reply:
This is actually a good point.